configure

```diff
@@ -1641,6 +1642,11 @@ maemo)
 	datadir=/opt/scummvm/share
 	docdir='${datarootdir}/doc/scummvm'
 	;;
+miyoo)
```

Comment:
As it's a variant on opendingux, could this be merged in opendingux-*?

Reply:
Makes no sense. It sets 3 vars and 2 of them are different from opendingux